examples/snakemake-freebayes-parallel.smk

```diff
@@ -14,7 +14,7 @@ chroms = [1,2,3]
 nchunks = 9
 
 bamlist = "path/to/bam.list"
-chunks = np.arange(1, nchunks)
+chunks = [i for i in range(1,nchunks+1)]
```

Comment:
Another way to write this is `chunks = list(range(1, nchunks+1))`, or maybe just `chunks = range(1, nchunks+1)` (if it works).

Reply:
Not sure why I put list comprehension there...
Snakemake does works with lists, I'm not sure if it works with ranges, so `list(range(...))` seems safest. Fixed in #690